README.md

```diff
@@ -122,14 +124,19 @@ iex> Siwe.validate(parsed, String.trim(sig))
 :true
 ```
 
-A trio of optimized helpers can be used to combine these steps:
+A pair of optimized helpers can be used to combine these steps:
 
-- `parse_if_valid_sig!` takes a SIWE message string and `signature`, then returns a parsed `Message` only if the `signature` matches.
+- `parse_if_valid_sig` takes a SIWE message string and `signature`, then returns a parsed `Message` only if the `signature` matches.
```

Comment:
Seems that `parse_if_valid_sig` has been removed too?

Reply:
Removed references to that